style.css

```diff
@@ -2261,7 +2261,7 @@ button.search-untoggle {
 
 .modal-search-form .search-field::-moz-placeholder {
 	color: inherit;
-	line-height: 8rem;
+	line-height: 128px;
```

Comment:
1rem equals 10px in Twenty Twenty, so you'll need to update this one accordingly.